tests/main/views/test_suppliers.py

```diff
@@ -1608,11 +1597,11 @@ def test_multiple_simultaneous_property_updates(
         assert data['frameworkInterest']['frameworkSlug'] == supplier_framework['frameworkSlug']
         assert data['frameworkInterest']['prefillDeclarationFromFrameworkSlug'] == "g-cloud-8"
 
-        with self.app.app_context():
-            supplier_framework2 = self._refetch_serialized_sf(data['frameworkInterest'])
-            assert data['frameworkInterest'] == supplier_framework2
-            audit = self._assert_and_return_audit_event(supplier_framework)
-            assert audit.data['update']['prefillDeclarationFromFrameworkSlug'] == "g-cloud-8"
+        supplier_framework2 = self._refetch_serialized_sf(data['frameworkInterest'])
+        assert data['frameworkInterest'] == supplier_framework2
+        audit = self._assert_and_return_audit_event(supplier_framework)
+        audit_id = audit.id
```

Comment:
Assign the attribute `audit_id` so we don't try to access a stale object later on.